internal/config/configs.go

```diff
@@ -18,21 +18,26 @@ type Config struct {
 	ServiceURLs struct {
 		MessagingService string
 		LoggingService   string
+		StudyService     string
 	}
 	UserDBConfig                      models.DBConfig
 	GlobalDBConfig                    models.DBConfig
 	Intervals                         models.Intervals
 	NewUserCountLimit                 int64
 	CleanUpUnverifiedUsersAfter       int64
 	ReminderToUnverifiedAccountsAfter int64
-	WeekDayStrategy                   utils.WeekDayStrategy
+	NotifyInactiveUsersAfter          int64
+	DeleteAccountAfterNotifyingUser   int64
+
+	WeekDayStrategy utils.WeekDayStrategy
 }
 
 func InitConfig() Config {
 	conf := Config{}
 	conf.Port = os.Getenv("USER_MANAGEMENT_LISTEN_PORT")
 	conf.ServiceURLs.MessagingService = os.Getenv("ADDR_MESSAGING_SERVICE")
 	conf.ServiceURLs.LoggingService = os.Getenv("ADDR_LOGGING_SERVICE")
+	conf.ServiceURLs.StudyService = os.Getenv("ADDR_STUDY_SERVICE")
```

Comment:
Could you please define this string as a const like some other env variable names are "ENV_..." 
Also maybe you could move the here hard coded other names to this const list as well, so it's consistent.